docs/development/environment.md

```diff
@@ -0,0 +1,11 @@
+# Enviroment
+
+This plugin contains both a server and web app portion. Read our documentation about the [Developer Workflow](https://developers.mattermost.com/extend/plugins/developer-workflow/) and [Developer Setup](https://developers.mattermost.com/extend/plugins/developer-setup/) for more information about developing and extending plugins.
+
+## Server
+
+Inside the `/server` directory, you will find the Go files that make up the server-side of the plugin. Within there, build the plugin like you would any other Go application.
+
+## Web App
+
+Inside the `/webapp` directory, you will find the JS and React files that make up the client-side of the plugin. Within there, modify files and components as necessary. Test your syntax by running `npm run build`.
```

Comment:
Consider changing the line "you will find the JavaScript and React files" to "you will find the JavaScript files" and not mention "React" at all since React is a Javascript framework, and React is implemented in JavaScript.

Reply:
I'm happy with that suggestion. Thank you!